setup.cfg

```diff
@@ -7,6 +7,7 @@ doc_files = README.txt
 
 
 [tool:pytest]
+testpaths=pydoctor/test
```

Comment:
won't this skip your doctests?

Reply:
ah you're not running inline doctests - they're all in `pydoctor/test/epydoc/*.doctest`

Reply:
Yeah, they are doctests in separate files because we inherited them from epydoc and the author thought doctests were a more convenient format than unit tests. That may have been true at the time, but now that we have pytest with less boilerplate and more detailed reporting of failures, I'm not sure doctests are still superior.

Reply:
sorry I don't have permission to resolve comments